src/BinSkim.Rules/PERules/BA2027.CompileUsingCurrentDialects.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Composition;
+using System.Linq;
+using System.Text;
+
+using Microsoft.CodeAnalysis.BinaryParsers;
+using Microsoft.CodeAnalysis.BinaryParsers.PortableExecutable;
+using Microsoft.CodeAnalysis.BinaryParsers.ProgramDatabase;
+using Microsoft.CodeAnalysis.IL.Sdk;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+namespace Microsoft.CodeAnalysis.IL.Rules
+{
+    [Export(typeof(Skimmer<BinaryAnalyzerContext>)), Export(typeof(ReportingDescriptor))]
+    public class CompileUsingCurrentDialects : WindowsBinaryAndPdbSkimmerBase
+    {
+        /// <summary>
+        /// BA2027
+        /// </summary>
+        public override string Id => RuleIds.CompileUsingCurrentDialects;
+
+        /// <summary>
+        /// /std setting enables supported C and C++ language features from the specified version of the C or C++ language standard. 
```

Comment:
>/std [](http://example.com/codeflow?start=12&length=4)

The '/std' setting

Also, separate this entire line in two lines.
the text is too long

Reply:
Sounds good will fix

Reply:
fixed